packages/changelog-linker/src/Github/GithubApi.php

```diff
@@ -113,15 +113,13 @@ public function isPullRequestMergedToBaseBranch(int $pullRequestId, string $base
     /**
      * @return mixed[]
      */
-    private function getPullRequestsSinceId(int $id, ?string $baseBranch = null): array
+    private function getPullRequestsSinceId(int $id, string $baseBranch): array
     {
         $pullRequests = [];
 
         for ($i = 1; $i <= self::MAX_PAGE; ++$i) {
             $url = sprintf(self::URL_CLOSED_PULL_REQUESTS, $this->repositoryName) . '&page=' . $i;
-            if ($baseBranch !== null) {
-                $url .= '&base=' . $baseBranch;
-            }
+            $url .= '&base=' . $baseBranch;
```

Comment:
should the branch name be escaped for use in a url?

Reply:
It might be, but Changelog Linker is now deprecated, see: https://github.com/symplify/symplify/issues/3027